package/scripts/httpfs.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import os
+from resource_management import *
+
+import logging
+
+logger = logging.getLogger()
+
+class HTTPFS(Script):
+  def install(self, env):
+    import params
+    env.set_params(params)
+    self.install_packages(env)
+
+
+  def configure(self, env):
+    import params
+    env.set_params(params)
+
+    Directory(params.app_conf_dir)
+
+    File(params.app_conf_dir + '/httpfs-site.xml',
+         content = Template('httpfs-site.xml.j2'),
+         owner = params.user,
+         group = params.group)
+
+    File(params.app_conf_dir + '/httpfs-env.sh',
+         content = StaticFile('httpfs-env.sh'),
```

Comment:
If you are talking about the file itself: that's the Python script that installs & starts the HTTPFS service.
We need this because we are using templates, so we have to use Python.

If you're talking about the config files (httpfs-env.sh & httpfs-log4j.properties): I understood we wanted to bring change only to httpfs-site. So we need to copy the other two as-is to the conf directory